inheritance/lib/src/main/java/inheritance/Review.java

```diff
@@ -0,0 +1,50 @@
+package inheritance;
+
+public class Review extends Restaurant {
```

Comment:
When you set up one class to extend another class, you're saying each the extending class has the same properties as a the extended class, plus a few more. In this case, that means each Review will have all the properties of a Restaurant, plus its own properties, rather than being associated with a Restaurant object.